_posts/2025-08-27-offer-multiple-ways-to-contact-you.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: post
+title: "I have a question, but I don't have WhatsApp!"
+permalink: /blog/2025/offer-multiple-ways-of-contact/
+author: Rian Rietveld
+categories: contact, multiple ways
+parent: Blog
+excerpt: "It’s better to offer multiple ways to get in touch, so people can choose what suits them. Think about your visitors first, not just what’s easiest for you.
+"
+date: 2025-08-27
+---
+You want to reach out to the customer service, and they say  *“Send us a WhatsApp message.”* What if you don't use WhatsApp, or cannot make a phone call, or use the chatbox? If you offer multiple contact options, you're leaving no people (customers) behind.
+{: .fs-6 .fw-300 }
+
+## Don’t make getting in touch their problem
+
+Let’s say someone wants to ask a quick question through your website, but the only way to do it is by sending a WhatsApp message.
+
+Sure, it’s convenient for you. One place to manage it all, and you reply when it suits you.
+
+Other examples:
```

Comment:
Make this line flow more smoothly:

"There are many similar examples of this type of problem:"

Reply:
I changed the text into
"There are many similar examples"